client/my-sites/marketing/business-tools/index.tsx

```diff
@@ -0,0 +1,216 @@
+/**
+ * External dependencies
+ */
+import { connect } from 'react-redux';
+import React, { Fragment, FunctionComponent } from 'react';
+import { useTranslate } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import { Button } from '@automattic/components';
+import { getSelectedSiteSlug } from 'calypso/state/ui/selectors';
+import MarketingBusinessToolsFeature from './feature';
+import MarketingBusinessToolsHeader from './header';
+import PageViewTracker from 'calypso/lib/analytics/page-view-tracker';
+import { recordTracksEvent as recordTracksEventAction } from 'calypso/state/analytics/actions';
+
+/**
+ * Images
+ */
+import quickbooksLogo from 'calypso/assets/images/illustrations/quickbooks-logo.svg';
+import evernoteLogo from 'calypso/assets/images/illustrations/evernote-logo.svg';
+import mondayLogo from 'calypso/assets/images/illustrations/monday-logo.svg';
+import benchLogo from 'calypso/assets/images/illustrations/bench-logo.svg';
+import todoistLogo from 'calypso/assets/images/illustrations/todoist-logo.svg';
+import streakLogo from 'calypso/assets/images/illustrations/streak-logo.png';
+import billcomLogo from 'calypso/assets/images/illustrations/billcom-logo.svg';
+
+/**
+ * Types
+ */
+import * as T from 'calypso/types';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+interface Props {
+	recordTracksEvent: typeof recordTracksEventAction;
+	selectedSiteSlug: T.SiteSlug | null;
+}
+
+export const MarketingBusinessTools: FunctionComponent< Props > = ( { recordTracksEvent } ) => {
+	const translate = useTranslate();
+
+	const handlePartnerClick = () => {
+		recordTracksEvent( 'calypso_marketing_business_boost_my_traffic_button_click' );
+	};
+
+	const handleQuickBooksClick = () => {
+		recordTracksEvent( 'calypso_marketing_business_quickbooks_button_click' );
+	};
+
+	const handleEvernoteClick = () => {
+		recordTracksEvent( 'calypso_marketing_business_evernote_button_click' );
+	};
+
+	const handleMondayClick = () => {
+		recordTracksEvent( 'calypso_marketing_business_monday_button_click' );
+	};
+
+	const handleTodoistClick = () => {
+		recordTracksEvent( 'calypso_marketing_business_todois_button_click' );
+	};
+
+	const handleBenchClick = () => {
+		recordTracksEvent( 'calypso_marketing_business_bench_button_click' );
+	};
+
+	const handleStreakClick = () => {
+		recordTracksEvent( 'calypso_marketing_tools_streak_button_click' );
+	};
+
+	const handleBillcomClick = () => {
+		recordTracksEvent( 'calypso_marketing_tools_billcom_button_click' );
+	};
+
+	return (
+		<Fragment>
+			<PageViewTracker path="/marketing/tools/:site" title="Marketing > Tools" />
+
+			<MarketingBusinessToolsHeader handleButtonClick={ handlePartnerClick } />
+
+			<div className="business-tools__feature-list">
+				<MarketingBusinessToolsFeature
```

Comment:
Same idea as above - we're using the same component with different data. Would be nice if we could abstract that data out and map over it to create each component. This would also make maintaining the feature set as easy as updating the data object.

Again, not a blocker and certainly not worth adjusting at this point. I'm mainly just noting some thoughts here. Everything still looks good! 😄